packages/page-accounts/src/modals/Create.tsx

```diff
@@ -387,13 +417,15 @@ function Create ({ className = '', onClose, onStatusChange, seed: propsSeed, typ
                       label={t<string>('secret derivation path')}
                       onChange={_onChangePath}
                       placeholder={
-                        seedType === 'raw'
-                          ? pairType === 'sr25519'
-                            ? t<string>('//hard/soft')
-                            : t<string>('//hard')
-                          : pairType === 'sr25519'
-                            ? t<string>('//hard/soft///password')
-                            : t<string>('//hard///password')
+                        pairType === 'ethereum'
+                          ? "m/44'/60'/0'/0/0"
```

Comment:
This should probably be a constant since it is duplicated in a number of places.

```js
const ETH_DEFAULT_PATH = "m/44'/60'/0'/0/0";
```